### PR TITLE
Hide minted by in mobile cards in drop page

### LINKED ISSUE #9229: Mobile cards in drop page
its overflowing hard 
image

- lets remove the "minted by + address" 
- there will only be the time and name on mobile devices

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Needs QA check

- @kodadot/qa-guild please review

## Context

- [x] Closes #9229

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=13Qx65nLd6SwdtjrRyuoEtp9CKXhF651xdHBPaXcvhwKm4N1)

## Screenshot 📸

- [] My fix has changed UI

Screen Shot 2024-02-01 at 10 16 33 PM

### CODE REVIEW

**Review by roiLeo — COMMENTED** [review]
Something strange on desktop view?
Screenshot 2024-02-02 at 08-33-13 KodaDot - NFT Market Explorer

**Comment by prachi00** [comment]
> @prachi00 please use PR template

earlier PR template use to come by default, did we remove that? @roiLeo

**Comment by roiLeo** [comment]
> earlier PR template use to come by default, did we remove that? @roiLeo

hmmm... https://github.com/kodadot/nft-gallery/tree/main/.github/PULL_REQUEST_TEMPLATE 

something wrong when opening PR?

**Comment by prachi00** [comment]
> Something strange on desktop view? Screenshot 2024-02-02 at 08-33-13 KodaDot - NFT Market Explorer

fixed

**Review by preschian — APPROVED** [review]

File: components/carousel/CarouselTypeLatestMints.vue [hunk 1]
```diff
@@ -14,7 +14,7 @@
           </nuxt-link>
 
           <div class="flex flex-col items-start">
-            <div class="flex justify-between items-center">
+            <div class="flex justify-between items-center hidden md:flex">
```
Comment (roiLeo) [inline components/carousel/CarouselTypeLatestMints.vue:17]: isn't first `flex` necessary since it's hidden ?

**Review by roiLeo — APPROVED** [review]

File: components/carousel/CarouselTypeLatestMints.vue (hunk 1 above)
Comment (prachi00) [reply components/carousel/CarouselTypeLatestMints.vue]: you are right, removed the duplicate  flex

**Comment by yangwao** [comment]
thanks!
pay 20 usd

**Pull request merged**